charts/posit-chronicle/README.md.gotmpl

```diff
@@ -0,0 +1,120 @@
+{{ template "rstudio.header" . }}
+
+{{ template "chart.versionBadge" . }}{{ template "chart.typeBadge" . }}{{ template "chart.appVersionBadge" . }}
+
+{{ template "rstudio.description" . }}
+
+{{ template "rstudio.disclaimer" . }}
+
+{{ template "rstudio.install" . }}
+
+## Usage
+
+This chart deploys only the Chronicle server and is meant to be used in tandem
+with the Workbench and Connect charts. To actually send data to the server, you
+will need to run the Chronicle agent as a sidecar container on all of your
+Workbench or Connect server pods by setting `pod.sidecar` in their respective
+`values.yaml` files.
+
+Here are some example Helm values to run the agent sidecar in Workbench:
+
+```yaml
+pod:
+  sidecar:
+    - name: chronicle-agent
+      image: posit-chronicle:latest
+      volumeMounts:
+      - name: CHRONICLE_PRODUCT_CLUSTER_ID
+        value: "posit-cluster-1"
+      - name: logs
+        mountPath: "/var/lib/rstudio-server/audit"
+      env:
+      - name: CHRONICLE_SERVER_ADDRESS
+        value: "http://chronicle-server.default.svc.cluster.local"
+```
+
+And here are some example Helm values for Connect, where we also pass a Connect
+API key from a Kubernetes Secret to unlock more detailed metrics:
+
+```yaml
+pod:
+  sidecar:
+    - name: chronicle-agent
+      image: posit-chronicle:latest
```

Comment:
It feels like putting the latest tag in the example might not be a great idea, could we template it to use the `appVersion` instead like our other helm charts?